README.md

```diff
@@ -157,11 +157,12 @@ Generates statistics for coverage using [BAMStats](http://bamstats.sourceforge.n
 ### 8. Mapped reads checkpoint
 [Checkpoints](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#data-dependent-conditional-execution) allow for data-dependent conditional execution, forcing Snakemake to re-evaluate the DAG at this point.
 
-Calculates the minimum number of reads required to meet the minimum coverage depth set for `varscan mpileup2snp` and checks the number of mapped reads from the bowtie2 alignment summary. The minimum number of reads required is calculated as:
+Determines if all segments within the reference genome has coverage summary generated from BAMStats.
+Then calculates the minimum number of reads required to meet the minimum coverage depth set for `varscan mpileup2snp` and checks the number of mapped reads from the bowtie2 alignment summary. The minimum number of reads required is calculated as:
 ```
 min_reads = (reference_genome_length * min_cov) / (raw_read_length)
 ```
-If `mapped_reads` is greater than or equal to `min_reads`, then continue with process to build consensus genome. If not, then stop the process for this sample/reference pair.
+If not all segments are represented or if `mapped_reads` is less than `min_reads`, then stop the process for this sample/reference pair.
 
```

Comment:
I think it's best if we squash the README changes to be in the same commit as the checkpoint changes.